matrix-appservice-irc/irc-registration.yaml

```diff
@@ -0,0 +1,20 @@
+id: 8932b3435f0dbb697307801fbcaedad2da2d29fa71c30d70ef433a21c0585db6
+hs_token: 711663d558bd4be0d41976a89347e86b14f7addcb15b661b14101e1f83196a53
+as_token: d81e0ccf5435cca28ee0140ccef48cf935810f95836865dc7014353c15e4a0fa
+namespaces:
+  users:
+    - exclusive: true
+      regex: '@.*_irc:matrix\.ocf\.berkeley\.edu'
+  aliases:
+    - exclusive: true
+      regex: '#.*:matrix\.ocf\.berkeley\.edu'
+  rooms:
+    - exclusive: false
+      regex: '!YAkpfBukqCvewVzbVb:matrix.ocf.berkeley.edu'
+    - exclusive: false
+      regex: '!zEhTjofIhaNlSpzxlN:matrix.ocf.berkeley.edu'
+url: 'http://localhost:9995'
```

Comment:
create a Kubernetes service for this port, then you can change this url to something like 'http://appservice-irc.app-matrix.svc.cluster.local:9995' which will let you move the two containers out of the same pod. no ingress is needed since this won't get hit from outside the k8s cluster